scene/2d/line_2d.cpp

```diff
@@ -95,6 +95,26 @@ float Line2D::get_width() const {
 	return _width;
 }
 
+void Line2D::set_curve(const Ref<Curve> &curve) {
```

Comment:
Parameters should be prefixed with `p_`, so `p_curve`.

Reply:
Done, I added the prefix in the other functions as well.